File: src/StudyProgram75/LevelOne/DayOne/RunningSumOf1dArray/GoLang/main.go
package main

func RunningSum(nums []int) []int {
	result := []int{}
	result = append(result, nums[0])
	size := len(nums)

	for i := 1; i < size; i++ {
		aux := result[i-1] + nums[i]

		result = append(result, aux)
	}

	return result
}

File: src/StudyProgram75/LevelOne/DayOne/RunningSumOf1dArray/NodeJs/main.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
const runningSum = function (nums) {
    let result = [];
    result.push(nums[0]);
    let size = nums.length - 1;

    for (let i = 1; i <= size; i++) {
        result[i] = result[i - 1] + nums[i];
    }

    return result;
};
